docs/guides/anomaly-detection-configuration/detection-delay.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "detection_delay"
+sidebarTitle: "detection_delay"
+---
+
+```
+detection_delay:
+  period: < time period > # supported periods: hour, day, week, month
+  count: < number of periods >
+```
+
+The duration for retracting the detection period.
```

Comment:
This is not clear enough in my opinion.
maybe something like: 
"There are cases in which the latest data should be excluded from the test. For example, this can happen because of scheduling issues- if the test is running before the table is populated for some reason. The detection delay is the period of time to ignore, after the detection period."
WDYT?

Reply:
i'll add it.